lib/modules/dosomething/dosomething_user/dosomething_user.module

```diff
@@ -698,6 +699,11 @@ function dosomething_user_user_insert(&$edit, &$account, $category = NULL) {
     $edit['name'] = $account->uid;
     $account->name = $account->uid;
   }
+
+  // Trigger an analytics event on registration
+  // @NOTE: The 'Authentication - Login' event will *also* be triggered here,
+  // since registered users are subsequently logged in to their new accounts.
+  dosomething_helpers_add_analytics_event('Authentication', 'Register');
```

Comment:
Ah, it looks like `user_insert` is intended more for database changes? If so, that makes sense to move it. :+1: